packages/shared-models/src/ymodels.ts

```diff
@@ -696,6 +696,25 @@ export class YBaseCell<Metadata extends models.ISharedBaseCellMetadata>
         { insert: ysource.toString() }
       ];
     }
+
+    // Other state changes
+    const stateChange: any = [];
+    modelEvent?.keysChanged.forEach(key => {
+      if (
+        ['metadata', 'execution_count', 'source'].find(value => value === key)
+      ) {
+        return;
+      }
+
+      const change = modelEvent.changes.keys.get(key);
+      stateChange.push({
+        name: key,
+        oldValue: change?.oldValue ? change!.oldValue : 0,
```

Comment:
Returning `0` as default value is dangerous because we don't know the type of the state element. I would suggest to stick with the same behavior than the one of execution count or metadata (aka `undefined` if unknown)

```suggestion
        oldValue: change?.oldValue,
```

Reply:
Thanks! That's a bad copy-paste